src/ensembl/src/content/app/browser/browser-bar/BrowserBar.tsx

```diff
@@ -119,6 +126,18 @@ export const BrowserBar: FunctionComponent<BrowserBarProps> = (
     props.toggleBrowserNav();
   };
 
+  const shouldShowBrowserTabs = () => {
+    if (
+      props.activeGenomeId &&
+      (props.isTrackPanelOpened ||
+        props.breakpointWidth === BreakpointWidth.LARGE)
+    ) {
+      return true;
+    } else {
+      return false;
+    }
+  };
```

Comment:
can we please immediately return the result of the equality check instead of checking whether the condition evaluates to true and then returning true?

E.g.

```
return props.activeGenomeId &&
      (props.isTrackPanelOpened ||
        props.breakpointWidth === BreakpointWidth.LARGE)
```

Reply:
There are too many conditions to evaluate here and that is why I thought it would be neater to put them in `if-else` clauses. Anyway, I'll return them with a ternary operator.

Reply:
@azangru As we discussed offline I misunderstood that you suggested to change this to a ternary operator. I'll return this sans any if statements and ternary operators.